lib/signs/realtime.ex

```diff
@@ -173,6 +187,18 @@ defmodule Signs.Realtime do
     end)
   end
 
+  defp add_extra_audio(%{extra_audio_sign_ids: extra_audio_sign_ids} = sign) do
+    Enum.map(extra_audio_sign_ids, fn sign_id ->
+      Signs.Realtime.get_sign_state(sign_id)
+      |> get_current_contents()
+    end)
+    |> then(fn content -> %Signs.Realtime{sign | extra_audio_content: content} end)
```

Comment:
Ideally we wouldn't store this in state, since it's only needed for the current iteration, and keeping state minimal should be a goal.